lib/diagnostic.rb

```diff
@@ -27,7 +37,12 @@
 # which returns the string "think think think".
 
 ##
-# your code here
+class Developer < Person
+
+  def initialize(solve_problems)
+    @solve_problems = "think think think"
+  end
+
```

Comment:
Your inheritance is correct, but you didn't create an instance method correctly.
```ruby
class Developer < Person
  def solve_problems
    "think think think"
  end
end
```